src/ciservices.rst

```diff
@@ -22,3 +22,16 @@ Circle can be manually added for each fork. Circle calls this "Adding a Project"
 This effectively amount to going to the `Add Projects <https://circleci.com/add-projects>`_
 page, finding the fork that you wish to enable, and clicking the "Build Project" button.
 This is not normally needed.
+
+Another issue is that Circle sometimes does not have permission to checkout the source code. You'll see a messsage like this::
```

Comment:
Maybe it is worth saying something like...

> If CircleCI lacks permissions to checkout the source code *in a PR* (as noted in the message below), here are some things you can try...

...or similar.

That should help clarify this is useful for user submitted PRs and not so much for builds on `master` for example.